test/PaymentReceiver-test.js

```diff
@@ -171,6 +171,7 @@ describe("PaymentReceiver", function () {
       .withArgs(admin.address, antGateId);
   });
 
+  /*
```

Comment:
can we put this back in? double check-ins should not be allowed (same user, same gate multiple times)